src/v2/Apps/ViewingRoom/ViewingRoomApp.tsx

```diff
@@ -48,16 +48,18 @@ const ViewingRoomApp: React.FC<ViewingRoomAppProps> = ({
   const getView = () => {
     if (!user) {
       return
-    } else if (viewingRoom.status === "closed") {
-      return <ViewingRoomClosed viewingRoom={viewingRoom} />
-    } else {
+    }
+
+    if (viewingRoom.status === "live") {
```

Comment:
So I think the only case we need to display content of the room is when the room is live I believe. I think it's more safe to rely on that case here.

This might change what we would currently see for Draft rooms since those were not guarded before and we would just display content. But I believe that is the correct way to access content going forward.

I believe the whole Force vr will be guarded against published rooms only. So that should not be an issue.